content/reference-transaction-format.md

```diff
@@ -659,22 +671,27 @@ These codes indicate that the transaction failed, but it could apply successfull
 
 ### tes Success
 
-The code `tesSUCCESS` is the only code that indicates a transaction succeeded. This does not always mean it did what it was supposed to do. (For example, an [OfferCancel][] can "succeed" even if there is no offer for it to cancel.) Success uses the numerical value 0.
+The code `tesSUCCESS` is the only code that indicates a transaction succeeded. This does not always mean it did what it was supposed to do. (For example, an [OfferCancel][] can "succeed" even if there is no offer for it to cancel.) The `tesSUCCESS` result uses the numerical value 0.
```

Comment:
Perhaps it is more fair to say that it 'may not have done what you expected', thant 'what it was supposed to do'. I would argue that it is doing exactly what it is supposed to do, by design. Suggest something like:
> This does not mean it succeeded with what you may have expected it to do.